<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageWindowUi</class>
 <widget class="QWidget" name="MessageWindowUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWebView" name="webView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
         <property name="renderHints">
          <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MessageEditorUi">
      <property name="windowTitle">
       <string>Message Editor</string>
      </property>
      <layout class="QVBoxLayout" name="mainLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="fontFrameLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="topFrame">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="topLayout">
              <property name="spacing">
               <number>1</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="fontStyleToolButton">
                <property name="text">
                 <string>Font Style</string>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/messagewindow/resources/images/messageeditor/font.png</normaloff>:/messagewindow/resources/images/messageeditor/font.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="emotionToolButton">
                <property name="text">
                 <string>Face</string>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/messagewindow/resources/images/messageeditor/face.png</normaloff>:/messagewindow/resources/images/messageeditor/face.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="insertImageToolButton">
                <property name="text">
                 <string>Insert Image</string>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/messagewindow/resources/images/messageeditor/insertimage.png</normaloff>:/messagewindow/resources/images/messageeditor/insertimage.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="sendFileToolButton">
                <property name="text">
                 <string>Send File</string>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/messagewindow/resources/images/messageeditor/file.png</normaloff>:/messagewindow/resources/images/messageeditor/file.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonScreenshot">
                <property name="text">
                 <string>Screenshot</string>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/messagewindow/resources/images/messageeditor/screenshot.png</normaloff>:/messagewindow/resources/images/messageeditor/screenshot.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="topSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>10</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="fontFrame">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="fontLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="leftMargin">
               <number>1</number>
              </property>
              <property name="topMargin">
               <number>1</number>
              </property>
              <property name="rightMargin">
               <number>1</number>
              </property>
              <property name="bottomMargin">
               <number>1</number>
              </property>
              <item>
               <widget class="QFontComboBox" name="fontComboBox">
                <property name="editable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="fontSizeComboBox">
                <property name="editable">
                 <bool>true</bool>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="fontBoldToolButton">
                <property name="text">
                 <string>Bold</string>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/messagewindow/resources/images/messageeditor/textbold.png</normaloff>:/messagewindow/resources/images/messageeditor/textbold.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="fontItalicToolButton">
                <property name="text">
                 <string>Italic</string>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/messagewindow/resources/images/messageeditor/textitalic.png</normaloff>:/messagewindow/resources/images/messageeditor/textitalic.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="fontUnderlineToolButton">
                <property name="text">
                 <string>Underline</string>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/messagewindow/resources/images/messageeditor/textunder.png</normaloff>:/messagewindow/resources/images/messageeditor/textunder.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="textColorToolButton">
                <property name="text">
                 <string>TextColor</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="bottomSapcerd">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>10</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'ææ³é©¿ç­å®½æ­£é»'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'宋体';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="acceptRichText">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="bottomLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="historyPushButton">
           <property name="text">
            <string>&amp;History</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="bottomSapcer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sendMsgPushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Send</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+Return</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
